tests/topotests/ospf_basic_functionality/test_ospf_lan.py

```diff
@@ -413,6 +413,7 @@ def test_ospf_lan_tc1_p0(request):
 
     build_config_from_json(tgen, topo_modify_change_ip, save_bkup=False)
 
+    clear_ospf(tgen, "r0")
```

Comment:
Here the test script wants to change the IP address of R0 so it should only reload the router R0. I think if we reload only the R0 config then this random problem won't occur. Since we are reloading all the routers' config, the test case is actually not testing the scenario "If a new router-ID with higher IP address comes in a stable env, it is not elected as DR". Therefore the behaviour was random.